frontend/src/System/Backup/BackupRow.js

```diff
@@ -30,47 +30,35 @@ class BackupRow extends Component {
 
   onRestorePress = () => {
     this.setState({ isRestoreModalOpen: true });
-  }
+  };
```

Comment:
A lot of unnecessary formatting here, please maintain the existing formatting.

Reply:
This was linted using the script in the package.json

Reply:
It passes linting, but in this case it's about keeping the consistency with rest of the repo. It looks like prettier (or similar) reformatted these lines.